#171602012 - Ender Yılmaz
#Bilişim Sistemleri Mühendisliği
#Elektronik Komponent Kayıt Kontrol Sistemi
from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_bobinler(object):
    def setupUi(self, bobinler):
        bobinler.setObjectName("bobinler")
        bobinler.resize(293, 190)
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("buton_resim/bobin.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        bobinler.setWindowIcon(icon)
        self.centralwidget = QtWidgets.QWidget(bobinler)
        self.centralwidget.setObjectName("centralwidget")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(20, 20, 111, 16))
        self.label.setObjectName("label")
        self.mgecirgenlik = QtWidgets.QLineEdit(self.centralwidget)
        self.mgecirgenlik.setGeometry(QtCore.QRect(130, 20, 151, 20))
        self.mgecirgenlik.setObjectName("mgecirgenlik")
        self.sarim = QtWidgets.QLineEdit(self.centralwidget)
        self.sarim.setGeometry(QtCore.QRect(130, 50, 151, 20))
        self.sarim.setObjectName("sarim")
        self.kesitalan = QtWidgets.QLineEdit(self.centralwidget)
        self.kesitalan.setGeometry(QtCore.QRect(130, 80, 151, 20))
        self.kesitalan.setObjectName("kesitalan")
        self.uzunluk = QtWidgets.QLineEdit(self.centralwidget)
        self.uzunluk.setGeometry(QtCore.QRect(130, 110, 151, 20))
        self.uzunluk.setObjectName("uzunluk")
        self.label_2 = QtWidgets.QLabel(self.centralwidget)
        self.label_2.setGeometry(QtCore.QRect(20, 50, 111, 16))
        self.label_2.setObjectName("label_2")
        self.label_3 = QtWidgets.QLabel(self.centralwidget)
        self.label_3.setGeometry(QtCore.QRect(20, 80, 111, 16))
        self.label_3.setObjectName("label_3")
        self.label_4 = QtWidgets.QLabel(self.centralwidget)
        self.label_4.setGeometry(QtCore.QRect(20, 110, 111, 16))
        self.label_4.setObjectName("label_4")
        self.hesapla_btn = QtWidgets.QPushButton(self.centralwidget)
        self.hesapla_btn.setGeometry(QtCore.QRect(230, 140, 51, 23))
        self.hesapla_btn.setObjectName("hesapla_btn")
        self.label_5 = QtWidgets.QLabel(self.centralwidget)
        self.label_5.setGeometry(QtCore.QRect(20, 140, 111, 16))
        self.label_5.setObjectName("label_5")
        self.enduktans_label = QtWidgets.QLabel(self.centralwidget)
        self.enduktans_label.setGeometry(QtCore.QRect(130, 140, 101, 16))
        font = QtGui.QFont()
        font.setBold(True)
        font.setWeight(75)
        self.enduktans_label.setFont(font)
        self.enduktans_label.setObjectName("enduktans_label")
        bobinler.setCentralWidget(self.centralwidget)
        self.statusbar = QtWidgets.QStatusBar(bobinler)
        self.statusbar.setObjectName("statusbar")
        bobinler.setStatusBar(self.statusbar)

        self.retranslateUi(bobinler)
        QtCore.QMetaObject.connectSlotsByName(bobinler)

    def retranslateUi(self, bobinler):
        _translate = QtCore.QCoreApplication.translate
        bobinler.setWindowTitle(_translate("bobinler", "Bobin Hesaplama"))
        self.label.setText(_translate("bobinler", "Manyetik Geçirgenlik :"))
        self.label_2.setText(_translate("bobinler", "Sarım Sayısı               :"))
        self.label_3.setText(_translate("bobinler", "Bobin Kesit Alan        :"))
        self.label_4.setText(_translate("bobinler", "Tel Uzunluğu             :"))
        self.hesapla_btn.setText(_translate("bobinler", "Hesapla"))
        self.label_5.setText(_translate("bobinler", "Endüktans                 :"))
        self.enduktans_label.setText(_translate("bobinler", "000 H"))